gr-utils/python/modtool/core/makeyaml.py

```diff
@@ -147,13 +147,16 @@ def _type_translate(p_type, default_v=None):
                 return 'raw'
         def _get_blockdata(fname_cc):
             """ Return the block name and the header file name from the .cc file name """
+            blockname_boolean = False
             blockname = os.path.splitext(os.path.basename(fname_cc.replace('_impl.', '.')))[0]
             fname_h = (blockname + '.h').replace('_impl.', '.')
+            if blockname.startswith(self.info['modname']+'_'):
```

Comment:
Yesterday I thought about a simplification of the syntax. You can just run `contains_modulename = blockname.startswith(....`

Reply:
Without any if-branching

Reply:
Thanks. I have made the changes.